### PR TITLE
[Feature/Bug(#133)] add icon to windows Exe and MSI

### PR DESCRIPTION
**Pull request by tensor-programming**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [x] Feature
- [ ] New Binding Issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [x] Yes. Issue #133
- [ ] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

Adds the Icon to the Exe and MSI in windows.

### CODE REVIEW

File: cli/tauri.js/templates/src-tauri/Cargo.toml [hunk 1]
```diff
@@ -19,6 +20,9 @@ phf = "0.7.24"
 includedir = "0.5.0"
 tauri = <%= tauriDep || `{ version = "0.2.0" }` %>
```
Comment (nothingismagick) [inline cli/tauri.js/templates/src-tauri/Cargo.toml]: Just noticed that phf and serde are still out of date.

Comment (tensor-programming) [reply cli/tauri.js/templates/src-tauri/Cargo.toml]: will update them.

**Comment by tensor-programming** [comment]
> do we still need phf and includedir here? because we already have them on tauri/cargo.toml

Honestly, couldn't tell you.  I don't see them being used in either of the files.

**Comment by lucasfernog** [comment]
> > do we still need phf and includedir here? because we already have them on tauri/cargo.toml
> 
> Honestly, couldn't tell you. I don't see them being used in either of the files. Ill remove them from one of the examples and see if it fails.

I believe @nothingismagick said he would do some cleanup on the examples, and we can remove those deps while he do it.

**Comment by tensor-programming** [comment]
> > > do we still need phf and includedir here? because we already have them on tauri/cargo.toml
> > 
> > 
> > Honestly, couldn't tell you. I don't see them being used in either of the files. Ill remove them from one of the examples and see if it fails.
> 
> I believe @nothingismagick said he would do some cleanup on the examples, and we can remove those deps while he do it.

I just removed them and it worked, so I will be removing them from the template.  Not sure what to do with the updater.rs file though, like I said, probably should be commented out or not pushed into a new project.

**Pull request merged**